.circleci/config.yml

```diff
@@ -14,6 +14,15 @@ python35_image: &python35_image circleci/python:3.5
 python27_image: &python27_image circleci/python:2.7
 ddtrace_dev_image: &ddtrace_dev_image datadog/dd-trace-py:latest
 datadog_agent_image: &datadog_agent_image datadog/docker-dd-agent:latest
+test_agent: &test_agent
+  - image: kyleverhoog/dd-trace-test-agent:latest
+    environment:
+      - SNAPSHOT_DIR=dd-trace-py/tests/snapshots
+      - SNAPSHOT_CI=1
+    # CircleCI does not provide a way to mount a volume between containers or a
+    # way to bake the snapshots into a custom image...
+    # So we have to clone the repo and snapshots into the container.
+    command: /bin/bash -c "git clone https://github.com/datadog/dd-trace-py/; cd dd-trace-py; git checkout << pipeline.git.branch >>; cd ..; java -jar app-standalone.jar"
```

Comment:
hmmm....

would we be better off using a different executor where we might be able to do this?

Reply:
Not quite sure what you mean 🤔. Can you clarify?

Reply:
this is gross, can we do something else?

Reply:
lol agreed. But see the commit history for my attempts at doing something else 😛. I'm totally open to anything other than this.